### PR TITLE
Fix UnmanagedMemoryStream test failure

### LINKED ISSUE #612: UmsSecurityTests.ChangePositoViaPointer failing
Every other run or so I keep hitting the following test failure in UnmanagedMemoryStreamTests

    UmsSecurityTests.ChangePositioViaPointer [FAIL] (TaskId:528)
        System.IO.IOException : An attempt was made to move the position before the beginning of the stream. (TaskId:528)
            Stack Trace: (TaskId:528)
            d:\git\corefx\src\System.IO.UnmanagedMemoryStream\src\System\IO\UnmanagedMemoryStream.cs(439,0): at System.IO.UnmanagedMemoryStream.set_PositionPointer(Byte* value) (TaskId:528)
            d:\git\corefx\src\System.IO.UnmanagedMemoryStream\tests\UmsSecurityTest.cs(34,0): at UmsSecurityTests.ChangePositioViaPointer() (TaskId:528)
                     Finished:    System.IO.UnmanagedMemoryStream.Tests (TaskId:528)

### PR DESCRIPTION
**Pull request by stephentoub**
A test was assuming that new byte[] allocations would come later in memory than the array allocation around which the stream was constructed.  When that assumption failed, so would the test.  This commit just rewrites the test to be explicit about the values being tested relative to the position of the fixed target array.

Fixes #612.

### CODE REVIEW

**Comment by jaredpar** [comment]
LGTM

File: src/System.IO.UnmanagedMemoryStream/tests/UmsSecurityTest.cs [hunk 1]
```diff
@@ -12,30 +12,25 @@ public class UmsSecurityTests
     public static void ChangePositioViaPointer()
     {
```
Comment (weshaggard) [inline src/System.IO.UnmanagedMemoryStream/tests/UmsSecurityTest.cs]: While your here might as well fix the typo in the test method name and add the 'n' in Position :)

**Comment by weshaggard** [comment]
LGTM thanks for fixing it @stephentoub 
cc @KrzysztofCwalina

File: src/System.IO.UnmanagedMemoryStream/tests/UmsSecurityTest.cs (hunk 1 above)
Comment (stephentoub) [inline src/System.IO.UnmanagedMemoryStream/tests/UmsSecurityTest.cs]: Ok :)

**Pull request merged**